docs/docs/sass.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Sass
```

Comment:
```suggestion
title: Using Sass in Gatsby
```
Might help discovery a little with SEO! I think the short title is fine in the sidebar.